openlibrary/plugins/upstream/forms.py

```diff
@@ -112,14 +112,6 @@ class RegisterForm(Form):
             maxlength="20",
             required="true",
         ),
-        Checkbox(
-            'ia_newsletter',
-            description=_(
-                'I want to receive news, announcements, and resources from the '
-                '<a href="https://archive.org/">Internet Archive</a>, the non-profit '
-                'that runs Open Library.'
-            ),
-        ),
         Checkbox(
```

Comment:
All `Checkbox` references need to be removed, as well as the `Checkbox` definition.